docs: add connector page for connecting external wallet connectors (#1547)
alchemyplatform/aa-sdk: docs/pages/react/login-methods/connect-to-other-wallet-providers.mdx

---
title: Seamlessly Connect with Your Favorite Web3 Wallets
description: How to connect to other wallet providers using Account Kit
---

> ## Seamlessly Connect with Your Favorite Web3 Wallets

Reviewer: ```suggestion
## Seamlessly Connect Existing EOA Wallets
```

Easily log in using popular wallet providers like **MetaMask**, **Rabby**, **Phantom**, and more.

> Thanks to our built-in wallet connectors, there's no need to integrate external solutions like RainbowKit.

Reviewer: ```suggestion
Our SDK supports external wallet connection out of the box, so you can seamlessly support both web2 users with social login and web3 users who want to bring their own EOAs.
```

> We automatically detect and display all wallet extensions available in the user's browser—ensuring a smooth and flexible experience.

Reviewer: ```suggestion
We support enabling users to bring their existing EOA in 2 ways:

1. Automatically detect and display all wallet extensions available in the user's browser
2. WalletConnect for all other EOAs

```

Reviewer: - Add a note and a link to how to set up the wallet connect config id.
- Could add a note to say if the user has no wallet extensions installed nothing will be displayed
- Add a note that our hooks will handle checking if the connected wallet is a smart wallet or an existing EOA. Transactions can be signed and sent in the same way, BUT callout that the EOA connected users dont get benefits like gas abstraction and batching - instead these EOAs will function as EOAs.

<Tip>
  With Account Kit, users can still connect to their existing wallets without
  sacrificing compatibility or convenience.
</Tip>

> <img src="/images/connect-to-other-wallet-providers" />

Reviewer 2: ⚠️ **[eslint]** <[jsx-a11y/alt-text](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/alt-text.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.
<!-- __reviewdog__:ChA5Njc0Njc1NmUyMGQyOTg2EgZyZGpzb24= -->

---
title: Seamlessly Connect with Existing EOA Wallets
description: How to connect to other wallet providers using Account Kit
---

## Seamlessly Connect with Existing EOA Wallets

Easily log in using popular wallet providers like **MetaMask**, **Rabby**, **Phantom**, and more.

Our SDK supports external wallet connection out of the box, so you can seamlessly support both web2 users with social login and web3 users who want to bring their own EOAs.

> We support enabling users to bring their existing EOA in 2 ways:

Reviewer 3: ```suggestion
Our SDK supports external wallet connection out of the box, so you can seamlessly support both Web2 users with social login and Web3 users who want to bring their own EOAs.
```
